aten/src/ATen/native/native_functions.yaml

```diff
@@ -5197,6 +5197,22 @@
     CPU, CUDA: max
     QuantizedCPU: max_quantized_cpu
 
+- func: maximum(Tensor self, Tensor other) -> Tensor
+  use_c10_dispatcher: full
+  variants: method, function
+
+- func: maximum.out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!)
+  dispatch:
```

Comment:
I don't think you need to be explicit about this dispatch. I believe it will automatically dispatch to `maximum_out`.

Reply:
Thanks for pointing that out, I have removed the redundant dispatch sections.